mmcv/utils/env.py

```diff
@@ -56,16 +56,20 @@ def collect_env():
         if CUDA_HOME is not None and osp.isdir(CUDA_HOME):
             try:
                 nvcc = osp.join(CUDA_HOME, 'bin/nvcc')
-                nvcc = subprocess.check_output(
-                    f'"{nvcc}" -V | tail -n1', shell=True)
+                nvcc = subprocess.check_output(f'"{nvcc}" -V', shell=True)
                 nvcc = nvcc.decode('utf-8').strip()
+                release = nvcc.rfind('release ')
+                build = nvcc.rfind('Build ')
+                nvcc = nvcc[release:build].strip()
             except subprocess.SubprocessError:
                 nvcc = 'Not Available'
             env_info['NVCC'] = nvcc
 
     try:
-        gcc = subprocess.check_output('gcc --version | head -n1', shell=True)
+        gcc = subprocess.check_output('gcc --version', shell=True)
         gcc = gcc.decode('utf-8').strip()
+        first_line = gcc.find('\n')
+        gcc = gcc[:first_line].strip()
```

Comment:
Hi, thanks for your contribution. mmcv compiled with `cl.exe` in windows, it could be better if `collect_env` can output the  
information of 'cl.exe'

Reply:
Usually there are multiple cl.exe on one machine (multiple VS version and cross-compliation tools) and cl is not avaliable in common shell environment. So it would be not an easy task and might even output misleading information. 

The main reason I need to patch this is, when I use `mmcv` *lite* version on Windows, it complains that no CUDA runtime is found, probably because of the missing `tail` and `head`.

Reply:
But on windows, mmcv does not compile with `gcc`, so it is confusing if it happens to output `mingw gcc` information. Should we choose how to export `gcc` information via `sys.platform`? Besides,  how does “no CUDA runtime is found” occurs?